test/built-ins/String/prototype/Symbol.iterator/prototype/next/missing-internal-slots.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.1.5.2.1 S 3
+description: >
+    If the `this` value does not have all of the internal slots of an String
+    Iterator Instance (21.1.5.3), throw a `TypeError` exception.
+---*/
+
+var iterator = ''[Symbol.iterator]();
+var object = Object.create(iterator);
+
+assert.throws(TypeError, function() {
+  object.next();
```

Comment:
Totally. Since these tests are submitted under the banner of "Import from V8", I'd like to hold off on including brand new tests for now. I've made the following note to come back to this:
> - Native iterators
>   - `next` methods should throw a `TypeError` if the context does not have all the internal slots of the appropriate iterator interface (e.g. [22.1.5.2.1 %ArrayIteratorPrototype%.next( )](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-%arrayiteratorprototype%.next), step 3). Specifically test this is the case even when such an object is in the context's prototype chain. See [this comment](https://github.com/tc39/test262/pull/210#discussion_r27394897) for context.